rpc_jobs/rpc_designate.yml

```diff
@@ -41,10 +41,17 @@
       - xenial:
           FLAVOR: "performance2-15"
           IMAGE: "Ubuntu 16.04.2 LTS prepared for RPC deployment"
+      - xenial_snapshot:
+          REGIONS: "DFW"
+          FALLBACK_REGIONS: ""
+          FLAVOR: "7"
+          BOOT_TIMEOUT: 1500
 
     scenario:
-      - "newton"
-      - "pike"
+      - newton:
+          IMAGE: "rpc-r14.9.0-xenial-swift"
```

Comment:
Who is responsible for incrementing these with each new release and are they aware?

Reply:
I haven't really gotten there yet -- I just want to add some jobs to start seeing how they run on a continual basis.